src/launcher/main.cpp

```diff
@@ -453,15 +479,24 @@ int main(int argc, const char** argv) {
             output = "jfr";
 
         } else if (arg == "--timeout" || arg == "--loop") {
-            params << "," << (arg.str() + 2) << "=" << args.next();
+            const char* value = args.next();
+            params << "," << (arg.str() + 2) << "=" << value;
+            // gprofiler-specific: borrow timeout parameter to configure socket timeout for fdtransfer action
+            timeout = atoi(value);
```

Comment:
I'd just add `--fdtransfer-timeout` instead of editing existing parameters into multiple meanings. Also less diff (only additions which are okay, but less modifications)

Reply:
👍